lib/compat/wordpress-5.9/class-gutenberg-block-template.php

```diff
@@ -9,7 +9,7 @@
 /**
  * Class representing a block template.
  */
-class WP_Block_Template {
+class Gutenberg_Block_Template {
```

Comment:
I've renamed this and the situation should be:
- If Gutenberg is running on WordPress 5.8, `Gutenberg_Block_Template` is used instead of WP_Block_Template`, since the latter is outdated in 5.8.
- If Gutenberg is running on WordPress 5.9, `WP_Block_Template` is used.

This should resolve any issues where `author` or `origin` might not be defined.